sdcm/stress/latte_thread.py

```diff
@@ -227,3 +229,29 @@ def _run_stress(self, loader, loader_idx, cpu_idx):
         return {}
         # TODOs:
         # 1) take back the report workload..3.0.8.p128.t1.c1.20231025.220812.json
+
+    def configure_event_on_failure(self, stress_event: LatteStressEvent, exc: Exception | Failure):
```

Comment:
It is the same as in `DockerBasedStressThread` - is it needed?

Reply:
It looks the same, but without it's explicit definition here we cannot use the custom `format_stress_cmd_error` function.

Reply:
uh, now I see `format_stress_cmd_error` is not part of the object.